tools/webpack/packages.js

```diff
@@ -73,7 +73,8 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 
 	const vendors = {
 		'lodash.js': 'lodash/lodash.js',
-		'wp-polyfill.js': '@babel/polyfill/dist/polyfill.js',
+		'wp-polyfill-core-js.js': 'core-js/stable/index.js',
```

Comment:
@desrosj, I need to do some research because it doesn't look like `core-js` has a distribution version but rather a long list of `require` calls ...